webapp/graphite/metrics/views.py

```diff
@@ -11,7 +11,9 @@
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License."""
-from urllib2 import urlopen
+import fnmatch
+import os
+import urllib2
```

Comment:
Note that `from urllib2 import urlopen` works fine for me. I've used `index.json` extensively in the past and it's never been an issue afaik.